docs/okp4-cognitarium-executemsg-definitions-binary.md

```diff
@@ -4,9 +4,9 @@
 undefined#/execute/definitions/Binary
 ```
 
-Binary is a wrapper around <code>Vec&lt;u8&gt;</code> to add base64 de/serialization with serde. It also adds some helper methods to help encode inline.
+Binary is a wrapper around Vec<u8> to add base64 de/serialization with serde. It also adds some helper methods to help encode inline.
```

Comment:
Sorry, I'm too late for this review 😅.

This will failed building documentation on Docusaurus since this will be interpreted as an HTML tag. 

I think there is an issue in the `sed` command :( 🥵.